server/static/css/main.css

```diff
@@ -827,5 +831,31 @@ input:-webkit-autofill:focus {
       margin: 0.4rem; }
     .table-footer button:last-child {
       margin-right: 0.5rem; } }
+.device-status-container {
+  background: #363430;
+  flex-direction: column;
+  border-radius: 0.5rem;
+  width: 100%;
+  margin: 0 5%; }
+  .device-status-container .device-status-content-header {
+    width: 100%;
+    height: 5rem;
+    padding: 0.5rem; }
+    .device-status-container .device-status-content-header button {
+      margin-right: 0.5rem; }
+    .device-status-container .device-status-content-header button:last-child {
+      margin-right: 0; }
+  .device-status-container .device-status-content {
+    height: 100%;
+    width: 100%;
+    flex-wrap: wrap; }
+    .device-status-container .device-status-content .chart-container {
```

Comment:
As it is now, charts look pretty narrow. I tried adjusting the width to 50em, which resulted in 3-1 chart composition on the page, but width of the charts was bigger and it seemed a bit more refined, yet keeping mobile version in pretty much the same shape.
![screenshot from 2019-03-06 12-13-48](https://user-images.githubusercontent.com/11870660/53869452-56ffd180-4009-11e9-8718-cdeb2bacbe6e.png)

It might be worth changing the width and extending the charts a bit to make it a bit less condensed.